index.js

```diff
@@ -1,3 +1,50 @@
 // javascript
 
-alert("Hello from index.js!");
+// Capture user's input on form submission - Set to variables
+//  Get the user's email address - userEmail
+//  Get the user's level - userLevel
+//  Get the user's hours of study - userHours
+let formData = document.querySelector("form");
+
+formData.addEventListener("submit", function (event) {
+  event.preventDefault();
+  console.log("Form submitted");
+});
+
+let userEmail;
+let userLevel;
+let userHours;
+
+console.log(userEmail);
+console.log(userLevel);
+console.log(userHours);
+
+// Validate the user's input - check with ifs. display errors by showing hidden text
+//  Check if the user has provided an email address - "E-mail address is required" is it an actual email
+//  Check if the user has selected a level - "You must select a course level"
+//  Check if the user has specified at least one hour of study - "You must select at least one hour per week of lessons"
+//  Check if the number of hours requested is within the allowed range - "You can only have a maximum of 5 hours per week with the beginner plan" / "You can only have a maximum of 10 hours per week with the advanced plan"
+function ValidateData() {
```

Comment:
Function names should be written in `camelCase`, where the first letter is lower cased.